bot.py

```diff
@@ -0,0 +1,24 @@
+"""
+Awesomebot entry point
+Mike Tung (seekheart)
+"""
+
+import discord
+import asyncio
+from settings import CREDS, PREFIX, logger
+
+
+
+def init_bot():
+    """Creates a discord client connection"""
+    return discord.Client()
+
+client = init_bot()
+
+@client.event
+async def on_ready():
+    """Logs the bot in for operations"""
+    print('logged in as {}'.format(client.user.name))
```

Comment:
We said we'd support 3.6+, yeah? Should use f-strings where we can then

Reply:
I've never used them, can you clone and make the changes? I would like to see.